core/bin/run

```diff
@@ -143,10 +168,22 @@ fi
 # Force Python scripts not to write .pyc files
 export PYTHONDONTWRITEBYTECODE=1
 
-# Run the script and append its output to its log file.
-echo "Running $SCRIPT_NAME (PID: $$)"
+# Make sure we have the SIMPLIFIED_* env vars
+source $CM_REPO_ROOT/../environment.sh
+
+#### Run the script and append its output to its log file. ###################
+
+log_info_to_stdout "$SCRIPT_NAME" "started"
+
 source $SIMPLIFIED_VENV/bin/activate && $FULL_SCRIPT_PATH "$@" >> $LOG_FILE 2>&1
 
+if [ $? -eq 0 ]; then
+  log_info_to_stdout "$SCRIPT_NAME" "completed"
+else
+  log_critical_to_stderr "$SCRIPT_NAME" "could not complete successfully, see $LOG_FILE"
+fi
+
```

Comment:
This adds a CRITICAL log entry (to CloudWatch and `/var/log/cron.log`) if the script exits non-zero, so we can support alarms on script failures if we want to. It doesn't tell you what actually went wrong, though you can trace it back by looking at the log file and comparing timestamps.

Future improvement would be to make sure each script emits either a simple summary message on success, or a detailed error for any non-zero exit, so we can capture the failure reason directly in the container output.